### ISSUE TITLE
feat(jans-linux-setup): changes to merge config-idp-plugin merged with config-saml-plugin

### ISSUE THREAD

**Issue by pujavs** [JanssenProject/jans#6922]
### 1. config changes
I have merged idp config with saml config and checked in `https://github.com/JanssenProject/jans/blob/jans-config-6869/jans-linux-setup/jans_setup/templates/jans-saml/jans-saml-config.json`

1.1. Update `jans-saml-config.json` with appropriate values 
Note: Mentioning to capture completeness for this change, as you might have already done when you implemented changes for [issues#6788](https://github.com/JanssenProject/jans/issues/6788)
[jans-saml-config.json](https://github.com/JanssenProject/jans/files/13549447/jans-saml-config.json)
Attached sample file used locally
Values that will change on setup is dependent on step#1
serverUrl": => KC server URL
"clientId": => KC clientId
"clientSecret": => KC client secret
"username": => KC server admin user
"password": KC server user pwd

### 2. Schema changes 
Modified schema for IDP as per requirement. Please review and make changes if needed
`https://github.com/JanssenProject/jans/blob/jans-config-6869/jans-linux-setup/jans_setup/schema/jans_schema.json`

### 3.  Directory structure
No additional change if already implemented as per [issues#6788](https://github.com/JanssenProject/jans/issues/6788)

### 4. Jetty setup changes
- Remove `idp-plugin` changes as this plugin no longer exits.
- `saml-plugin` jar has been renamed as `kc-saml-plugin.jar`.  Please fo related changed
-  **Copy new jar From:** Copy renamed jar from maven repo ->  kc-saml-plugin.jar-1.0.21-SNAPSHOT.jar from maven repo -> [https://maven.jans.io/maven/io/jans/jans-config-api/plugins/idp-plugin//](https://maven.jans.io/maven/io/jans/jans-config-api/plugins/kc-saml-plugin/1.0.21-SNAPSHOT/)
**To:** /opt/jans/jetty/jans-config-api/ custom/libs/
Example location for 1.0.15-SNAPSHOT->https://maven.jans.io/maven/io/jans/jans-config-api/plugins/idp-plugin/1.0.21-SNAPSHOT/
- Rename jar to `kc-saml-plugin.jar` in /opt/jans/jetty/jans-config-api/ custom/libs/

- jans-config-api.xml changes:  
    Remove entry for `/opt/jans/jetty/jans-config-api/custom/libs/idp-plugin.jar`
    Rename entry for to to `kc-saml-plugin.jar` to  `kc-saml-plugin.jar` 

Note:
Jans config api changes for idp-plugin endpoints is still in branch [jans-config-6869](https://github.com/JanssenProject/jans/tree/jans-config-6869
PR raised to merge the same => [6921](https://github.com/JanssenProject/jans/pull/6921)

Related issues:
https://github.com/JanssenProject/jans/issues/6869

**Comment by devrimyatar** [comment]
Implemented in https://github.com/JanssenProject/jans/pull/6895
